aws/solutions/lambda-backed-cloudformation-custom-resources/get_vpc_main_route_table_id/Routetable.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import print_function
+import json
+import boto3
+import urllib
+from botocore.vendored import requests
+
+
+SUCCESS = "SUCCESS"
+FAILED = "FAILED"
+
+def send(event, context, responseStatus, responseData, physicalResourceId=None, noEcho=False):
+    responseUrl = event['ResponseURL']
+
+    print(responseUrl)
+
+    responseBody = {}
```

Comment:
Avoid repetition and use a dict literal here.

Reply:
modified